tracer/src/Datadog.Trace/TracerManagerFactory.cs

```diff
@@ -150,21 +150,29 @@ private static IDogStatsd CreateDogStatsdClient(ImmutableTracerSettings settings
                 }
 
                 var statsd = new DogStatsdService();
-                if (AzureAppServices.Metadata.IsRelevant)
+                if (settings.Exporter.MetricsTransport == Vendors.StatsdClient.Transport.TransportType.NamedPipe)
                 {
-                    // Environment variables set by the Azure App Service extension are used internally.
+                    // Environment variables set by the Azure App Service extension are used internal to the config api.
```

Comment:
not sure what "used internal to the config api" is talking about, but then I'm not sure what it meant before either so 🤷‍♂️ 😄

Reply:
Yeah reading this back gives me a headache.